src/gds-upgrade/sass/_tables.scss

```diff
@@ -128,3 +128,20 @@
     }
   }
 }
+
+// custom width
```

Comment:
Just to know which parts refer to "old" tables and which ones are for "new" tables I think we should move these classes to the _tables.scss file inside components folder

Reply:
Also, could you add the two classes you used for the services dashboard here too (make names consistent) and update them in the services template then?
`<th scope="col" class="govuk-table__header width-35">Service</th>
                                <th scope="col" class="govuk-table__header width-65">Description</th>`

Reply:
Done!